integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/manowar/EnvironmentLegacyAuthzGetTest.java

```diff
@@ -0,0 +1,72 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.manowar;
+
+import javax.inject.Inject;
+
+import org.testng.annotations.Test;
+
+import com.sequenceiq.environment.api.v1.environment.model.response.EnvironmentStatus;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.Status;
+import com.sequenceiq.it.cloudbreak.actor.CloudbreakActor;
+import com.sequenceiq.it.cloudbreak.client.CredentialTestClient;
+import com.sequenceiq.it.cloudbreak.client.EnvironmentTestClient;
+import com.sequenceiq.it.cloudbreak.client.FreeIpaTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.environment.EnvironmentTestDto;
+import com.sequenceiq.it.cloudbreak.dto.freeipa.FreeIpaTestDto;
+import com.sequenceiq.it.cloudbreak.dto.telemetry.TelemetryTestDto;
+import com.sequenceiq.it.cloudbreak.testcase.authorization.AuthUserKeys;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest;
+
+public class EnvironmentLegacyAuthzGetTest extends AbstractE2ETest {
```

Comment:
this test logic almost the same as in `EnvironmentLegacyAuthzGetTest ` in `authorization` package, you should make here a common part also to remove duplications